src/test/java/uk/gov/hmcts/reform/bailcaseapi/domain/handlers/presubmit/UploadBailEvidenceDocumentHandlerTest.java

```diff
@@ -164,17 +164,6 @@ void should_add_new_evidence_to_the_case_when_no_evidence_documents_exist() {
         verify(bailCase, times(1)).write(BAIL_EVIDENCE_WITH_METADATA, allEvidenceDocuments);
     }
 
-    @Test
```

Comment:
Please change the test to not do anything if evidence is not present instead of removing. Check - it shouldn't throw exception, or change the bailCase object.

Reply:
Added test case to handle null entry.